src/views/catalog/wizard/tabs/scripts/components/DynamicSSHKeyInjectionWizard.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from 'react';
+import { Updater } from 'use-immer';
+
+import { WizardDescriptionItem } from '@catalog/wizard/components/WizardDescriptionItem';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  convertYAMLUserDataObject,
+  getCloudInitData,
+  getCloudInitVolume,
+} from '@kubevirt-utils/components/CloudinitModal/utils/cloudinit-utils';
+import { DynamicSSHKeyInjection } from '@kubevirt-utils/components/DynamicSSHKeyInjection/DynamicSSHKeyInjection';
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import {
+  getCloudInitConfigDrive,
+  getCloudIntPropagationMethod,
+} from '@kubevirt-utils/components/SSHSecretSection/utils/utils';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getAccessCredentials, getVMSSHSecretName, getVolumes } from '@kubevirt-utils/resources/vm';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { PopoverPosition } from '@patternfly/react-core';
+
+type DynamicSSHKeyInjectionWizardProps = {
+  updateVM: Updater<V1VirtualMachine>;
+  vm: V1VirtualMachine;
+};
```

Comment:
you can remove these props and get them from useVMWizardContext directly

Reply:
You right, thank you ! :)